### ISSUE TITLE
[bug] openobserve upgrade to 0.10.9-rc4, the histrogram is not valid

### ISSUE THREAD

**Issue by oslet** [openobserve/openobserve-grafana-plugin#12]
the reason is: sql_mode is remove, pls support, Thank you
![Uploading 2024-08-15_22-28.png…]()

**Comment by hengfeiyang** [comment]
@omkarK06 we removed `aggs` support from search API. we need fire two request for logs list and histogram.

**Comment by erikwoo** [comment]
wish to have histogram + 1

**Comment by hengfeiyang** [comment]
Yes, we know, but we have no energy on this case, maybe someone can help fix it.

**Comment by elaurensx** [comment]
I've also ran into this and it's such a shame as it prevents me from visualizing ingested logs on my Grafana dashboards. Any directions as to where one needs to start looking to work on this issue?

**Comment by hengfeiyang** [comment]
i think the problem is API change, it don't compatible Grafana plugin.

**Comment by gaby** [comment]
@hengfeiyang Is this a grafana or OpenObserve api change?

**Comment by hengfeiyang** [comment]
O2 API change caused this issue.
